build.sbt

```diff
@@ -206,6 +217,12 @@ def playJsonTests(scalacVersion: String, includePlayVersion: String, includeScal
     case (Scala_2_13, Play_2_7) => Seq(
       `play27-json-ops-213`
     )
+    case (Scala_2_12, Play_2_8) => Seq(
+      `play28-json-ops-212`
+    )
+    case (Scala_2_13, Play_2_8) => Seq(
+      `play28-json-ops-213`
+    )
     case _ => Seq()
```

Comment:
Q: Is this an invalid state? Maybe it should throw or something if it's invalid?

Reply:
I think so, but I'm not certain.